Makefile

```diff
@@ -11,13 +11,17 @@ GORUN = env GO111MODULE=on go run
 libzkp:
 	cd $(PWD)/rollup/circuitcapacitychecker/libzkp && make libzkp
 
-nccc_geth: ## geth without circuit capacity checker
-	$(GORUN) build/ci.go install ./cmd/geth
+libzstd:
+	@wget https://github.com/scroll-tech/da-codec/releases/download/v0.0.0-rc0-ubuntu20.04/libzktrie.so
+	@wget https://github.com/scroll-tech/da-codec/releases/download/v0.0.0-rc0-ubuntu20.04/libscroll_zstd.so
+
+nccc_geth: libzstd ## geth without circuit capacity checker
+	@LD_LIBRARY_PATH="$(LD_LIBRARY_PATH):$(PWD)" CGO_LDFLAGS="-L$(PWD) -Wl,-rpath=$(PWD)" $(GORUN) build/ci.go install ./cmd/geth
 	@echo "Done building."
 	@echo "Run \"$(GOBIN)/geth\" to launch geth."
 
-geth: libzkp
-	$(GORUN) build/ci.go install -buildtags circuit_capacity_checker ./cmd/geth
+geth: libzkp libzstd
+	@LD_LIBRARY_PATH="$(LD_LIBRARY_PATH):$(PWD)" CGO_LDFLAGS="-L$(PWD) -Wl,-rpath=$(PWD)" $(GORUN) build/ci.go install -buildtags circuit_capacity_checker ./cmd/geth
```

Comment:
GORUN remove `env GO111MODULE=on`

Reply:
removed in [cf3785c](https://github.com/scroll-tech/go-ethereum/pull/726/commits/cf3785cfe58c7ad404b584128b3fd6f7bd101473). it's geth's original Makefile, so it's also OK not to remove it.